libs/utils/src/postgres_backend.rs

```diff
@@ -165,12 +165,8 @@ pub fn is_socket_read_timed_out(error: &anyhow::Error) -> bool {
 
 // Truncate 0 from C string in Bytes and stringify it (returns slice, no allocations)
```

Comment:
the comment still mentions `Bytes`

Reply:
Thanks.

---

To be honest, I find this function very suspicious. It's not at all clear to me why some of the so-called "C-strings" do not end with a null terminator. But that's a story for another day.